ci/scripts/unit.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+pushd {.Name}
```

Comment:
I saw the PR to update the template of the error, think we need to update this here too
```suggestion
pushd dp-bulletin-api
```

Reply:
Thanks, spotted this too and have a commit waiting to be pushed that will fix this. 👍